assets/js/atomic/blocks/product-elements/rating-counter/index.tsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import { registerBlockType } from '@wordpress/blocks';
+import { isExperimentalBuild } from '@woocommerce/block-settings';
+
+/**
+ * Internal dependencies
+ */
+import metadata from './block.json';
+import edit from './edit';
+import sharedConfig from '../shared/config';
+import { supports } from './support';
+import { BLOCK_ICON } from './constants';
+
+if ( isExperimentalBuild() ) {
+	registerBlockType( metadata, {
+		...sharedConfig,
+		ancestor: [
+			'woocommerce/all-products',
+			'woocommerce/single-product',
+			'core/post-template',
+			'woocommerce/product-template',
+		],
+		icon: { src: BLOCK_ICON },
+		supports,
+		edit,
+	} );
+}
```

Comment:
It seems like `icon` is duplicated/overridden between:
1. `block.json`
2. `constants.tsx`.

Also, you can move `ancestor` to `block.json` as well to keep it in one place, what do you think?